### PR TITLE
Fix misspelled hyperlink to posts archive

### PR DESCRIPTION
**Pull request by plamen**

### CODE REVIEW

File: layouts/index.html [hunk 1]
```diff
@@ -27,5 +27,5 @@ <h3 class = 'article_title'>{{ $value.Title }}</h3>
   {{- if and (eq $index (add $size -1)) (gt $size 1) }}</div>{{ end }}
   {{- end }}
 </div>
-<a href = '{{ absURL (printf "post/%s" "") }}' class = 'post_nav'><span class = 'post_next'>View Archive</span></a>
+<a href = '{{ absURL (printf "posts/%s" "") }}' class = 'post_nav'><span class = 'post_next'>View Archive</span></a>
```
Comment (onweru) [inline layouts/index.html]: @plamen, thanks for contributing. This commit will break things while working with the [exampleSite](https://github.com/onweru/newsroom/tree/master/exampleSite/content). As you can see, the posts directory is named `post` in that case. To solve this issue for good, perhaps there is a way not to hard code the directory into the path. I will check and proceed accordingly.

File: layouts/partials/footer.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <footer class = 'footer wrap pale'>
-  <p>&copy; <span class = 'year'></span> {{ .Site.Title }}</p>
+  <p>&copy;&nbsp;<span class = 'year'></span>&nbsp;{{ .Site.Title }}</p>
```
Comment (onweru) [inline layouts/partials/footer.html]: This looks good. Thanks.

File: layouts/index.html (hunk 1 above)
Comment (plamen) [reply layouts/index.html]: done, removed the commit

**Comment by onweru** [comment]
Awesome @plamen, thanks!

I will follow up on the `post` directory issue soon.

**Pull request merged**